<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_new_patient_ui</class>
 <widget class="QDialog" name="add_new_patient_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DrStrange v.0.1-pre-release</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-science">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI';&quot;&gt;This is add new patient page&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientSSN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientEmail">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientMobilePhone">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientAddress1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>427</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientAddress2">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>457</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientAddressCity">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>487</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientAddressState">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>517</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientAddressZip">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>547</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Allergy History</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_allergy">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lab History</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lab">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diagonistics</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_diagonistics">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addPatientSave">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientDOB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DOB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>DOB:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zip Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="addPatientEmailLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Email Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLabel" name="FirstName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>83</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SSN#</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mobile Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>427</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Address 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>457</y>
     <width>83</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Address 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LastName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="MiddleName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Name: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientFirstName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientLastName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientMiddleName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Work Phone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientWorkPhone">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Home Phone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addPatientHomePhone">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="addPatientResult">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
